### ISSUE TITLE
management console UI should ask for password instead of using browser's built in basic auth form

### ISSUE THREAD

**Issue by slimsag** [sourcegraph/sourcegraph#1685]
The basic auth form feels janky, we should just have the react app ask for the password when you visit the page instead. Then it would feel more integrated into the app, etc.

cc @francisschmaltz who suggested this + for designing how this form would look.

The form should only ask for the password (not username), and have a link to management console documentation (so the user can figure out where to get the password). No other information is needed on this page, but we could e.g. include the Sourcegraph logo etc.

**Comment by ryan-blunden** [comment]
Is there a way to pre-fill the username to be `sourcegraph` in the meantime? I did not find it obvious that the username could be anything until I re-read the docs.

**Comment by slimsag** [comment]
Nope, unfortunately there is not. We do send a server-side message indicating that the password can be anything, but unfortunately most browsers no longer allow display of those.

**Comment by ryan-blunden** [comment]
Is it possible to change the the text "Sign in" on the login window to "Sign in (username is sourcegraph) or something like that?

**Comment by slimsag** [comment]
We already request browsers do that, but only Firefox respects our request. So no, unfortunately. There is a separate issue somewhere for moving away from basic auth prompt to our own one, which would be far nicer.

**Comment by sqs** [comment]
Backlogging. I think we all agree it would be nice to have this, but I haven't heard of any customers being blocked by this. That makes it lower priority than other things @slimsag 	is working on.
